### PR TITLE
Support option fileTime for incremental migration

### PR DESCRIPTION
**Pull request by ruhan1**
Support only scan files after speicified time point.

### CODE REVIEW

File: src/main/java/org/commonjava/migrate/pathmap/MigrateOptions.java [hunk 1]
```diff
@@ -359,6 +377,33 @@ private boolean validateOptions()
         return true;
     }
 
+    private static final SimpleDateFormat FILE_DATE_FORMAT = new SimpleDateFormat( "yyyyMMddHH" );
```
Comment (ligangty) [inline src/main/java/org/commonjava/migrate/pathmap/MigrateOptions.java]: Should use this SimpleDateFormat as local var as it is not thread-safe.

**Review by ligangty — COMMENTED** [review]
LGTM except one thread-safe issue

File: src/main/java/org/commonjava/migrate/pathmap/MigrateOptions.java (hunk 1 above)
Comment (ruhan1) [reply src/main/java/org/commonjava/migrate/pathmap/MigrateOptions.java]: Ok. UPdated.

**Comment by ruhan1** [comment]
@ligangty I also record the global last modified time so next time we have the time as input. pls view the second commit.

**Review by ligangty — APPROVED** [review]
LGTM

**Pull request merged**